File: 1 - Multiples of 3 and 5/rust/src/main.rs
fn multiples_of_3_and_5(number: i32) -> i32 {
    let mut sum  : i32 = 0;
    for i in 0..number {
        if i % 3 == 0 || i % 5 == 0 {
            sum += i;
        }
    }
    sum
}

fn main() {
    let result = multiples_of_3_and_5(1000);
    println!("{}", result);
}

File: 1 - Multiples of 3 and 5/main.ts
function multiples(num : number) : number {
    let sum : number = 0;
    for(let i=0;i<num;i++){
      if(i % 3 == 0 || i % 5 == 0){
        sum+=i;
      }
    }
    return sum;
  }
  
  multiples(1000);

